src/utils/tests/perms.test.js

```diff
@@ -0,0 +1,54 @@
+import * as permUtils from '../perms';
+
+describe('perms', () => {
+  let mockMember = {
+    fetch: jest.fn().mockImplementation(() => {}),
+    hasPermission: jest.fn().mockImplementation(() => false),
+    roles: {
+      cache: {
+        each: jest.fn()
+      },
+      highest: { name: 'Contributor' }
+    }
+  };
+  const message = {
+    member: {}
+  };
+
+  test('getMemberFromMessage', () => {
+    const member = permUtils.getMemberFromMessage(message);
+    expect(member).toEqual({});
+  });
+
+  // test('findRole', () => {
```

Comment:
should this be uncommented or removed?

Reply:
Yeah, I'll remove. I was debating whether or not to keep trying to test it